bundles/SeoBundle/src/Model/Redirect.php

```diff
@@ -264,7 +264,7 @@ public function isRegex(): bool
 
     public function setRegex(?bool $regex): static
     {
-        $this->regex = $regex ? (bool) $regex : null;
+        $this->regex = $regex ? $regex : null;
```

Comment:
We could also change this?
```suggestion
        $this->regex = $regex ?: null;
```

Reply:
Why can't it be false?

Reply:
```suggestion
        $this->regex = $regex;
```